lms/services/jstor/service.py

```diff
@@ -124,7 +161,7 @@ def _api_request(self, path_template, doi, params=None) -> requests.Response:
         if "/" not in doi:
             doi = f"{self.DEFAULT_DOI_PREFIX}/{doi}"
 
-        url = self._api_url + path_template.format(doi=quote(doi, safe="/"))
+        url = self._api_url.rstrip("/") + path_template.format(doi=quote(doi, safe="/"))
```

Comment:
Kind of incidental, but our URLs look broken in live as they have double slashes in them. It works fine, but looks like it could be a source of errors (especially when dealing with 404s).